assets/css/custom.css

```
@@ -1017,9 +1014,9 @@ html.is-nav-carry .animated {
   html,
   .html,
   .site-bg__poster {
-    /* Narrow and portrait: same cover-width as landscape — never crop sides. */
-    --bg-size: 100% auto;
-    background-size: 100% auto;
+    /* Same cover fill as desktop — layer bleed still overflows the clip. */
+    --bg-size: cover;
+    background-size: cover;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)